tests/mm_cataloguer/test_associate_ensemble.py

```diff
@@ -34,7 +34,7 @@ def index_test_files(Session):
         'data/tiny_hydromodel_gcm.nc',
         'data/tiny_gcm_climo_monthly.nc',
     ]
-    filenames = [resource_filename('modelmeta', f) for f in test_files]
+    filenames = [(files("modelmeta") / f).resolve() for f in test_files]
     for filename in filenames:
         with CFDataset(filename) as cf:
```

Comment:
The tests end up printing out a *ton* of errors from python-netcdf4:

```
Exception ignored in: 'netCDF4._netCDF4.Dataset.__dealloc__'
AttributeError: __getattribute__
AttributeError: __getattribute__
```

which I think ultimately stems from the fact that `nchelpers` inherits from `netCDF4.Dataset` [even though it shouldn't](https://github.com/Unidata/netcdf4-python/issues/788). Fixing that is beyond the scope of this PR, but it is something that we should address in `nchelpers`.